gnowsys-ndf/gnowsys_ndf/ndf/mapping_files/attribute_map.json

```diff
@@ -0,0 +1,6178 @@
+        "This programme is based on the theme of the environmental protection to encourage children about a healthy , green and clean environment.", 
+        "TThis art work by 'Santosh Jain' made in digital media is based on a poem 'Todti Patthar', written by Suryakant Tripathi 'Nirala'. This image shows a labourer woman splitting stones on the bank of the river Ganga", 
+        "This program is based on the importance of Photosynthesis", 
+        "This program is based on an old Japanese folk tale. Through a story of a singing bird, the program highlights the values of freedom and liberty.", 
+        "The focal message of this song is to continuously move on despite hurdles.", 
+        "The programme is based on the Indian classical music with practical demonstration.", 
+        "The program tells about the simple scientific ways and methods of the disposal of garbage and encourages children to keep their surroundings and environment clean. The classification of garbage is also explained", 
+        "Kathak  Parichay  video series developed by CIET, NCERT", 
+        "This program gives a glimpse of the post Vedic period. The program describes about the then culture, economy and political conditions.", 
+        "This program gives information about the various organs of animals and  the human body", 
+        "This program is based on the story of a boy named Gappu. The program orients children about health issues.", 
+        "This program is based on the poems of Hindi textbook Bal bharati", 
+        "This programme tells children about some interesting methods to add different numbers.", 
+        "This program is taken from Buddha Charit that describes about the great enlightenment of Mahatma Buddha.", 
+        "The programme teaches children about the formation of shadows.", 
+        "This program is a talk on the subject areas of Geography. The program mainly describes about the preservation of water as a very precious resource.", 
+        "This program is based on an old folk tale of the Munda tribe it describes about the values of trees in our lives", 
+        "This program describes about the virtues of nature which are available to us free of cost.", 
+        "This programme presents a song on agriculture. The manner of singing is also taught.", 
+        "This program raises environmental concerns by telling that because of the environmental pollution the greenery is lost.", 
+        "The programme presents a song of patriotism and communicates the message that under any condition we should not bow down to the forces that destroys the fabric of the nation.", 
+        "This programme offers new and interesting ways to understand addition in mathematics. Some interesting activities are carried out in this programme to enable children to understand the concept of addition.", 
+        "The programme describes about the Yoga and instructs children about some of the A
```

Comment:
This file and folder seems to be created from script. I think it has got added/committed by mistake. Please remove this file and folder.
> `git rm <folder name or file name>`